integration/presto/src/main/java/org/apache/carbondata/presto/PrestoFilterUtil.java

```diff
@@ -171,134 +170,89 @@ else if (colType.equals(DecimalType.createDecimalType(carbondataColumnHandle.get
    * @return
    */
   static Expression parseFilterExpression(TupleDomain<ColumnHandle> originalConstraint) {
-    ImmutableList.Builder<Expression> filters = ImmutableList.builder();
 
     Domain domain;
+    Expression finalFilters = null;
```

Comment:
Can you provide comments to explain the below Expression?  and why colExpression need put "final"?

Expression finalFilters = null;
final Expression colExpression = new ColumnExpression(cdch.getColumnName(), coltype);
Expression colValueExpression = null;
Expression rangeExpression = null;

Reply:
1. finalFilters => this one is the final expression for the table,  this variable is returned by the method after combining all the column filters (colValueExpression).
2. colExpression => we have used final to make sure that value of this variable doesn't get change during the execution of iteration for respective column.
3. colValueExpression => this is the combination of multiple rangeExpression for a single column in case of multiple filters on single column else this is equal to rangeExpression
4.rangeExpression => this is generated for each range of column i.e. lessThan, greaterThan, there can be multiple ranges for a single column.

Reply:
1. I mean, please put these detail explanation inside the code in your pr
2. colExpression, just column's name and data type, why may change ?

Reply:
sure, adding comments in code